mscore/scoreview.cpp

```diff
@@ -1821,13 +1821,15 @@ void ScoreView::cmd(const QAction* a)
                 if (avsOmr) {
                     avsOmr->config().setIsHideAll(!avsOmr->config().isHideAll());
                     getAction("avsomr-hide-all")->setChecked(avsOmr->config().isHideAll());
+                    this->update(); //! NOTE force redraw (especially needed for MacOS)
                 }
           }
           else if (cmd == "avsomr-hide-recognized") {
                 auto avsOmr = _score->masterScore()->avsOmr();
                 if (avsOmr) {
                     avsOmr->config().setIsHideRecognized(!avsOmr->config().isHideRecognized());
                     getAction("avsomr-hide-recognized")->setChecked(avsOmr->config().isHideRecognized());
+                    this->update(); //! NOTE force redraw (especially needed for MacOS)
```

Comment:
needed for maxOS only? If so it might better get `#ifdef Q_OS_MAC`?

Reply:
I found that I didn’t redraw the view on the MacOS after hiding the menu, but I think this is also useful for other platforms. It also allows you to see changes in rendering immediately after clicking on a menu item, and not after hiding the menu panel.

Reply:
Ok